packages/core/src/WebPlayer.ts

```diff
@@ -27,7 +27,7 @@ export default class WebPlayer extends EventEmitter {
   async load(src: string, autoplay = false) {
     this.video.muted = autoplay;
     this.video.autoplay = autoplay;
-    this.video.setAttribute("playsinline", "");
+    this.video.setAttribute('playsinline', '');
```

Comment:
To reduce risks for merge conflicts, try to keep only code changes relevant to this feat/issue in the PR.